fractions/monitor/src/main/java/org/wildfly/swarm/monitor/runtime/InstallMonitorFilter.java

```diff
@@ -47,10 +42,11 @@
 
     @Override
     public void prepareArchive(Archive<?> archive) {
-        JARArchive jarArchive = archive.as(JARArchive.class);
-
-        Asset resource = new ClassAsset(HealthResponseFilter.class);
-        ArchivePath location = new BasicPath(PATH_CLASSES, AssetUtil.getFullPathForClassResource(HealthResponseFilter.class));
-        jarArchive.add(resource, location);
+        try {
```

Comment:
I think this can be removed by creating a module.xml for health-api.jar and then adding that module to module.conf with export=true, similar to the jaxrs client api cdi extension we have.